notes/transcript15.md

```diff
@@ -213,108 +213,228 @@ Igor: I guess the signature won't signatures won't match but we can do it a shot
-Ricardo: So maybe we could support a Java version that would be a good option because Java is a is a really clear written language and like mostly understood widely. So I am volunteering myself of updating Etherium J to support new whisper and then to start making rising up mail server or something. So but I'm doing that in free time. So maybe this could this effort could help in that the protocol definition because having these implementations just make it to reach
-Oskar: sorry Ricardo. I want to make sure we don't stray too much into protocol and disorientation so on. I think go back to topic with whisper finishing and so on. Like what's the specific steps we want to do here. I think one is just roughly capturing the current protocol and so these types of changes would have sort of carry more weight. So if there's gonna be some some kind of breaking compatible or whatever that is sort of formatted in in a way that's outside of the code base. First help in this fixed repository. Do people generally go with that?
+Oskar: There was another point if it was a good point it was brought up which is the lack of engagement. Just process speaking I think that partitioning topics is like a great way to optimize the bad way from the second. It is a reasonable thing to do and it seems like something that generally could be done local sort of local form decision-making within the Swarm. I think that concern might be when it comes to something that breaks the protocol so that it impacts sort of more groups of people and I think if that is made more clear or if we sort of we want to separate out all of this and have it be a sort of different cultural process and different technical process where we're changing the protocol and we need some form of consensus perhaps in these calls, perhaps in some other form. That's something we're going to work on over the next few weeks and months to sort of we have some haphazard documentation but bring it more together and also create this specs repository and then maybe we when we have questions like these, then we can bring them up in the core dev call as of come prepare with a proposal for changing some, the protocol in some way and then we can discuss it. I think that would separate out the things there are sort of swarm implementation specifics for certain trade-offs versus things that impact the entire protocol. And if people have any thoughts on that.
```

Comment:
"Just process speaking I think that partitioning topics is like a great way to optimize the bad way from the second."

Looks like this phrase is not clear, maybe you can clarify it @oskarth in case it was mistranscribed?

Reply:
I don't remember tbh

Reply:
can probably be removed and still be somewhat understandable